Update readme files (#29)
ASFHyP3/OpenData: asf-event-data/README.md

to a range of disaster events. 

This repository contains a number of different prefixes, some supporting specific response efforts, and others 
supporting general disaster response services. 

## Table of Contents
1. [Introduction](#introduction)
2. [Disaster Support Services](#disaster-support-services)
3. [Disaster Event Prefixes](#disaster-event-prefixes)
4. [Data Products](#data-products)
5. [Accessing the Data](#accessing-the-data)
6. [License](#license)
7. [Manifest](#manifest)

## Introduction

Synthetic Aperture Radar (SAR) data is a powerful tool for monitoring and assessing disaster events. 
The Alaska Satellite Facility (ASF) has curated this collection of SAR data products, making it accessible 
to the public through this AWS Open Data Bucket. The data covers a range of disaster events, providing valuable insights for researchers, scientists, and emergency response teams.

## Disaster Support Services

Some of the prefixes include content generated in support of ongoing monitoring services. The data can 
be accessed through interactive web applications, which automatically provide access to new data products as 
they are added to the bucket. These prefixes include: 

> - RTC_services

Reviewer: ```suggestion
- image-services/RTC_services
```

- HKHwatermaps

### RTC Services

The contents of the RTC_services prefix are Radiometric Terrain Corrected (RTC) Sentinel-1 products generated 
to support the [NASA Earth Science Applied Science Disasters](https://appliedsciences.nasa.gov/what-we-do/disasters) 
program. These RTC products cover the east and west coasts of the United States and the entire state of Alaska, 
and can be accessed through image services hosted by ASF. This allows the data to be available immediately to 

> disaster response personnel as soon as possible without any additional processing. 

Reviewer: RTC_services includes these file extensions. Probably worth noting the water extent maps in this paragraph.

```
VH.tif
VV.tif
VV.tif.xml
WM.tif
WM_HAND.tif
dem.tif
rgb.tif
```

Author: @asjohnston-asf I ended up revamping this section a fair bit.

  and the processing workflow
- **WM.tif** - Surface water extent raster
- **WM_HAND.tif** - Height Above Nearest Drainage, used to generate the WM product
- **dem.tif** - Copy of the Digital Elevation Model used to process the RTC product 
  ([Copernicus GLO-30 DEM](https://dataspace.copernicus.eu/explore-data/data-collections/copernicus-contributing-missions/collections-description/COP-DEM) 
  with an ellipsoid correction applied)
- **rgb.tif** - False-color [RGB Decomposition](https://github.com/ASFHyP3/hyp3-lib/blob/main/docs/rgb_decomposition.md), 
  combining both VV and VH returns

Note that the WM (surface water extent) products have only been 
[validated in the Hindu Kush Himalayan region](https://www.mdpi.com/2072-4292/16/17/3244). 
While they are available for other areas, they may not perform well in all cases. Use caution when referencing these 
products, as areas with very dry or frozen soils may be classified as surface water. 

These products can be explored and accessed through interactive web applications, which are regularly updated to 
include new acquisitions. They are hosted in the following prefixes:
- image-services/RTC_services
- HKHwatermaps

> 

Reviewer: ```suggestion

The image-service-overviews prefix contains ESRI Cloud Raster Format (CRF) mosaics supporting ArcGIS image services of these data sets.

```

Author: I would probably add this to the RTC section below this one.

### RTC Services

The files in the image-services/RTC_services prefix are generated to support the 
[NASA Earth Science Applied Science Disasters](https://appliedsciences.nasa.gov/what-we-do/disasters) 
program. These products cover the east and west coasts of the United States and the entire state of Alaska. 

ASF has published image services for the VV, VH, and RGB products hosted under this prefix. These image services 
allow the data to be available immediately to disaster response personnel, without any additional processing. 
They can be explored interactively in 
[this web map](https://asf-daac.maps.arcgis.com/home/webmap/viewer.html?webmap=3dd8d25559db4ba6aa0e1b6e8cb5d39a).

New Sentinel-1 acquisitions are processed to RTC and transferred to this prefix every three hours, and image 
services are updated daily.

> 

Author: ```suggestion

Mosaic-level overviews in Cloud Raster Format (CRF) are generated for these services and hosted in the `image-service-overviews` prefix.

```

## License

> <--[TODO: Change to our usual license]-->
> 
> The data in this AWS Open Data Bucket is provided freely and openly under the XX policy. You may use this data for any purpose you would like.

Reviewer: ```suggestion
The data in this AWS Open Data Bucket is provided freely and openly under the [Creative Commons Zero (CC0) 1.0 Universal License](https://creativecommons.org/publicdomain/zero/1.0/). You may use this data for any purpose you would like.
```
